meta-lmp-base/files/lmp-group-table

```diff
@@ -69,5 +70,7 @@ systemd-bus-proxy:x:991:
 systemd-timesync:x:992:
 systemd-resolve:x:993:
 systemd-journal:x:994:
+pulse-access:x:995:
+pulse-rt:x:996:
```

Comment:
Please use 973 and 974 for them, as I want to make sure that the last ones are used/reserved for systemd.

Reply:
done!